jekyll/_cci2/orb-author.md

```diff
@@ -9,279 +9,289 @@ version:
 
-When designing your own orbs, make sure your orbs meet the following requirements:
+#### 4) You will be asked a short series of questions to configure and setup your orb such as your [namespace]({{site.baseurl}}/2.0/orb-concepts/#namespaces) and orb name.
+{:.no_toc}
 
-* Orbs should always use `description`. - Be sure to explain usage, assumptions, and any tricks in the `description` key under jobs, commands, executors, and parameters.
-* Match commands to executors - If you are providing commands, try to provide one or more executors in which they will run.
-* Use concise naming for your orb - Remember that use of your commands and jobs is always contextual to your orb, so you can use general names like "run-tests" in most cases.
-* Required vs. optional parameters - Provide sound default values of parameters whenever possible.
-* Avoid job-only orbs - Job-only orbs are inflexible. While these orbs are sometimes appropriate, it can be frustrating for users to be unable to use the commands in their own jobs. Pre and post steps when invoking jobs are a workaround for users.
-* Parameter `steps` are powerful - Wrapping steps provided by the user allows you to encapsulate and sugar things like caching strategies and other more complex tasks, providing a lot of value to users.
+In the background the `orb init` command will be copying and customizing the [Orb Project Template](https://github.com/CircleCI-Public/Orb-Project-Template) based on your inputs. There is a detailed `README.md` file within each directory that contains helpful information specific to that directory.
 
-Refer to [Reusing Config]({{ site.baseurl }}/2.0/reusing-config/) for details and examples of commands, executors and parameters in orbs.
+You will also be asked for the remote git repository URL which we obtained in step 1. Ensure you are using the `HTTPS` URL.
 
-When developing your own orb, you may find it useful to write an inline orb. The section below describes how you can write your own inline orb.
+#### 5) When prompted, in a separate terminal, push the changes up to Github.
+{:.no_toc}
 
-### Writing Inline Orbs
+During the setup process, the `orb init` command will take several steps to prepare your automated orb development pipeline. The modified template code produced by the CLI must be pushed to the repository before the CLI can continue and automatically follow your project on CircleCI.com successfully.
 
-Inline orbs can be handy during development of an orb or as a convenience for name-spacing jobs and commands in lengthy configurations, particularly if you later intend to share the orb with others.
+```bash
+$ git push origin master
+```
 
-To write inline orbs, place the orb elements under that orb's key in the `orbs` declaration in the configuration. For example, if you want to import one orb and then author inline for another, the orb might look like the example shown below:
+For technical reasons, we can not automatically push this code.
```

Comment:
"for technical reasons" = "due to permissions" ?

Reply:
Sort of, but nothing to do with CircleCI, its a limitation of the library we are using for git in the CLI and being platform agnostic. @gmemstr could maybe provide more specifics. Maybe we dont have to mention it and just instead give them the instructions.

Reply:
Yeah let's just not say anything :-) 👍